src/use-case/hybrid-vector-search.md

```diff
@@ -0,0 +1,251 @@
+$$
+D[1\times 256]=\left[
+ \begin{matrix}
+   0.342 & 1.774 & 0.087 & 0.321 & 0.664 & 0.870 & 0.001 & \cdots & 0.543 & 0.999
+  \end{matrix}
+  \right]
+$$
+
+
+Dense vectors and sparse vectors are both used to represent features of text chunks, but they have significant differences. A single dimension in a dense vector embedding does not mean anything, as it is too abstract to determine its meaning. However, when we take all the dimensions together, they provide the semantic meaning of the input text.
+
+In contrast, sparse vectors are related to the vocabulary, with dimensions equal to the vocabulary size. Therefore, the dimensions can reach tens of thousands or even hundreds of thousands, but most elements are zero. Each element of a sparse vector corresponds to the weight of a word in the text fragment.
+
+In general, words with higher weights indicate a higher importance in the text. It is much easier for users to analyze or visualize the relationship between the text and the vocabulary with sparse vectors. While for dense vectors, some dimension reduction algorithms are required for visualization, such as:
+
+- [Principal component analysis (PCA)](https://en.wikipedia.org/wiki/Principal_component_analysis)
+- [T-distributed stochastic neighbor embedding (T-SNE)](https://en.wikipedia.org/wiki/T-distributed_stochastic_neighbor_embedding)
+- [Uniform Manifold Approximation and Projection (UMAP)](https://arxiv.org/abs/1802.03426)
+
+|                      | Dense Vector          | Sparse Vector               |
+| -------------------- | --------------------- | --------------------------- |
+| Dimension            | Hundreds to thousands | Tens of thousands           |
+| Dimension Meaning    | Nothing               | Word Weight                 |
+| Interpretability     | Hard                  | Easy                        |
+| Usage                | Vector Distance       | Vector Distance / Visualize |
+| Semantic sensitivity | High                  | Low                         |
+
+Sparse vectors are generally less semantically sensitive than dense vectors. They have a harder time capturing the relationships between words, especially for synonyms and related terms.
+
+For example, the words "rocket" and "SPACE X" are both related to space exploration. Dense vectors are more likely to reveal this similarity and assign a closer vector distance to the corresponding texts of these words.
+
+## How to create a vector embedding?
+
+A vector embedding is the internal representation of input data in deep learning models, also known as embedding models. Most embedding models, such as `text-embedding-3-small`, are dense models, that only output dense embddings. We need special models designed for sparse embedding.
```

Comment:
```suggestion
A vector embedding is the internal representation of input data in deep learning models, also known as embedding models. Most embedding models, such as `text-embedding-3-small`, are dense models, that only output dense embedding. We need special models designed for sparse embedding.
```

Reply:
Thanks, I have fixed the passage by DeepL.